plugins/dell-k2/fu-dell-k2-rmm.c

```diff
@@ -127,7 +83,7 @@ fu_dell_k2_rmm_init(FuDellK2Rmm *self)
 	fu_device_add_icon(FU_DEVICE(self), "dock-usb");
 	fu_device_add_flag(FU_DEVICE(self), FWUPD_DEVICE_FLAG_UPDATABLE);
 	fu_device_add_flag(FU_DEVICE(self), FWUPD_DEVICE_FLAG_SIGNED_PAYLOAD);
-	fu_device_add_private_flag(FU_DEVICE(self), FU_DEVICE_PRIVATE_FLAG_USE_PROXY_FOR_OPEN);
+	fu_device_add_flag(FU_DEVICE(self), FWUPD_DEVICE_FLAG_INSTALL_SKIP_VERSION_CHECK);
```

Comment:
this really feels like a slippery slope -- we're not actually going to be doing "did it work" checks if we use this flag.